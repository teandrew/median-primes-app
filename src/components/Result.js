import React from "react";

const Result = ({ medianPrimes }) => (
  <div style={{ fontSize: "20px" }}>
    <p>The median prime(s): {medianPrimes.toString()}</p>
  </div>
);

export default Result;
