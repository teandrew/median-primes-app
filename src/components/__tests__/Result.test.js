import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Result from "../Result";

const mockPrimes = [2, 3];

describe("Results Component", () => {
  it("renders", () => {
    const wrapper = shallow(<Result medianPrimes={mockPrimes} />);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
