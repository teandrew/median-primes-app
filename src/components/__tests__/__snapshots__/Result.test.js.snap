// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results Component renders 1`] = `
<div
  style={
    Object {
      "fontSize": "20px",
    }
  }
>
  <p>
    The median prime(s): 
    2,3
  </p>
</div>
`;
